src/CalcManager/Header Files/CalcUtils.h

```diff
@@ -3,11 +3,13 @@
 
 #pragma once
 
-bool IsOpInRange(uintptr_t op, uint32_t x, uint32_t y);
-bool IsBinOpCode(uintptr_t opCode);
+typedef uintptr_t opcode_t;
```

Comment:
Let's go with `OpCode`.  We should use PascalCase for user-defined types.  We should also leave `_t` for standard library types.

Reply:
Maybe this is a personal preference, but I try to write with the `using OpCode = uintpr_t` syntax in new code because it's easier for new readers to parse and understand.

Reply:
Sure, I'll address both of these today. The `using` syntax is much nicer, I always get ripped `typedef`s wrong and I'm not sure why I haven't used it here in the first place.

Reply:
Updated